assets/grafana/dashboard-definitions.yaml

```diff
@@ -6122,4 +6122,347 @@ items:
   metadata:
     name: grafana-dashboard-k8s-resources-pod
     namespace: openshift-monitoring
+- apiVersion: v1
+  data:
+    persistentvolumesusage.json: |-
```

Comment:
Do we want to ship this dashboard? I still haven't tested it, as I still do not have these metrics.
I suggest we ignore this for now as well, just like we do with others:
https://github.com/openshift/cluster-monitoring-operator/blob/bf7f82146647419af396f0bdf08b6c17ed6f10eb/jsonnet/main.jsonnet#L78